siac/walletcmd.go

```diff
@@ -116,6 +109,42 @@ Run 'wallet send --help' to see a list of available units.`,
 		Long:  "View wallet status, including the current balance and number of addresses.",
 		Run:   wrap(walletstatuscmd),
 	}
+
+	walletUnlockCmd = &cobra.Command{
+		Use:   "unlock \"[password]\"",
+		Short: "Unlock the wallet",
+		Long:  "Decrypt and load the wallet into memory",
+		Run:   wrap(walletunlockcmd),
+	}
+
+	walletInitCmd = &cobra.Command{
+		Use:   "init",
+		Short: "Initilize and encrypt a new wallet",
+		Long: `Generate primary seed used for the wallet and encrypt using that
```

Comment:
wording is a bit awkward. I would write:
```
Generate a new wallet from a seed string, and encrypt it.
The seed string and encryption password will be returned.
```

also, the above should be "Initialize"